wxt_cidrbot/git_webhook_handler.py

```diff
@@ -108,6 +110,15 @@ def webhook_request(self, event):
             elif event_action == 'review_requested':
                 self.send_review_message(installation_id, json_string, False, None)
 
+        elif x_event_type == 'pull_request_review':
+            state = json_string['review']['state']
+            state = state.lower()
+
+            self.logging.debug("JSON DATA: %s", json.dumps((event["body"])))
```

Comment:
Just checking - there's nothing in this data like password, tokens, credentials, etc. that shouldn't be logged, right?